gito/cli.py

```
@@ -157,6 +157,10 @@ def cmd_answer(
         default=None,
         help="GitHub Pull Request number"
     ),
+    aux_files: list[str] = typer.Option(
+        default=None,
+        help="Auxiliary files that might be helpful"
+    )
```

Ruff (0.12.2)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)